app/upgrades/v6/constants.go

```diff
@@ -0,0 +1,20 @@
+package v6
+
+import (
+	"cosmossdk.io/store/types"
+	"github.com/MANTRA-Chain/mantrachain/v6/app/upgrades"
+)
+
+const (
+	// UpgradeName defines the on-chain upgrade name.
+	UpgradeName = "v6.0.0"
```

Comment:
it's only for manninet right? just a bit duplicate from v6.0.0-rc0

Reply:
I guess we don't need to keep the testnet upgrade handler.